Data Dictionary

```diff
@@ -0,0 +1,62 @@
+ Data Dictionary for GNU Radio:
+ 
+ 
+ 
+Flow graph: A graph containing at least one source & sink block through which data flows or referred to as a digraph of blocks.
+
+
+Blocks: allows only one signal processing operation at a time with an input port on one end & an output port on the other end- exception for source and sink blocks.
+
```

Comment:
Stream	A continuous flow of consecutive items
IO signature	A description of a blocks input and output ports